MojioSDK/Client/Core/RestClient.swift

```diff
@@ -113,16 +113,19 @@ open class RestClient {
     
     public let sessionManager: SessionManager
     internal var keychainManager: KeychainManager
+    private let arrayEncoding: URLEncoding.ArrayEncoding
     
     public init(
         clientEnvironment: ClientEnvironment,
         sessionManager: SessionManager = SessionManager.default,
-        keychainManager: KeychainManager? = nil) {
+        keychainManager: KeychainManager? = nil,
+        arrayEncoding: URLEncoding.ArrayEncoding = .noBrackets) {
```

Comment:
@andriy-morgunov-epam  Do we need the value `.noBrackets` as default value?

Reply:
Why we can not use an optional value?

Reply:
Leaving that for future work as it was incidental to debug mode.